City-of-Helsinki/kukkuu-admin
# KK-593 | Add message creation view

## Description

In this PR I add the creation view for messages. This means that I integrate the creation API and provide the view layer for the user to interact with the API.

I added a few new abstractions
- `<KukkuuCreationPage />`
- `<KukkuuCreationToolbar />`
- `<EventSelect />`
- `<MessageForm />`
    - I'll hopefully be able to reuse this with the edit view

## Context

After these changes, users will be able to create new messages by using the admin UI.

[KK-593](https://helsinkisolutionoffice.atlassian.net/browse/KK-593)

## How Has This Been Tested?

I've tested these changes by hand.

## Manual Testing Instructions for Reviewers

As a logged in admin user

1. Access message list by using the left navigation menu
1. Click on + New"
1. Expect to be taken into the view in the screenshot
1. Expect all field labels to be shrinked
1. Expect receivers to be all by default
1. Expect event to be all by default
1. Expect to be able to choose between these receivers: all, invited, enrolled, arrived, has subscribed to notifications
1. Expect to be able to choose from all events
1. Expect to be able to provide a title and a message
1. Expect that title and message fields change when you change language from the language tab
1. Expect form submission to fail and errors to be shown if one, many or all of the following fields are validated incorrectly
    - receiver: required, one of the value in the dropdown
    - subject: required
    - message: required
1. After a successful submit, expect to be taken into the list view
1. Expect cancel button to take you back to the list view


## Screenshots

<img width="1680" alt="Screenshot 2020-10-21 at 14 22 40" src="https://user-images.githubusercontent.com/9090689/96713199-e9a0e000-13a8-11eb-9854-0c79a4e5153e.png">
